server.js

```diff
@@ -68,13 +68,26 @@ app.use(
     })
 )
 
+const requireAuth = (req, res, next) => {
+    if (req.user) {
+        next();
+    } else {
+        res.redirect('/login');
```

Comment:
We don't send redirects on backend for json API. The should be res.status(403).send('Not Authorized')